spec/drafts/v2.1.1/stix-v2.1.1.adoc

```diff
@@ -10513,12 +10500,12 @@ Adversaries who are motivated by coercion are often forced through intimidation
 |[stixliteral]#dominance#
 |A desire to assert superiority over someone or something else.
 
-Adversaries who are seeking dominance over a target are focused on using their power to force their target into submission or irrelevance. Dominance may be found with ideology in some state-sponsored attacks and with notoriety in some cyber vandalism-based attacks.
+Adversaries who are seeking dominance over a target are focused on using their power to force their target into submission or irrelevance.
 
 |[stixliteral]#ideology#
 |A passion to express a set of ideas, beliefs, and values that may shape and drive harmful and illegal acts.
 
-Adversaries who act for ideological reasons (e.g., political, religious, human rights, environmental, desire to cause chaos/anarchy, etc.) are not usually motivated primarily by the desire for profit; they are acting on their own sense of morality, justice, or political loyalty.
+Adversaries who act for ideological reasons (e.g., environmental, desire to cause chaos/anarchy, etc.) are not usually motivated primarily by the desire for profit; they are acting on their own sense of morality, justice, or political loyalty.
```

Comment:
`Adversaries who act for ideological reasons are not usually motivated primarily by the desire for profit; they are acting on their own sense of morality, justice, or political loyalty` I would remove the example to keep a kind of neutrality.